e2e/test_module.py

```diff
@@ -13,15 +14,18 @@ def db():
 class TestConstants:
     ABSOLUTE_TOLERANCE = 1e-3
 
+    CLEANUP = "cleanup"
     EXCEPTION = "exception"
     INPUT_FILE = "input.cyp"
     OUTPUT = "output"
+    ONLINE_TEST = "_test"
+    SETUP = "setup"
     QUERY = "query"
+    QUERIES = "queries"
     TEST_DIR_ENDING = "_test"
+    TEST_DIR_ONLINE_PREFIX = "test_online"
```

Comment:
I think we should rename this, because name doesn't give away correct structure:
test_dir
   subtest_dir
   subtest_dir

TEST_DIR could be named MODULE_DIR_TESTING_SUFFIX
and TEST_DIR_ONLINE_PREFIX could be named MODULE_ONLINE_SUBTEST_DIR_PREFIX
and then for static tests we could have MODULE_STATIC_ONLINE_SUBTEST_DIR_PREFIX = "test"

Reply:
This way we are giving away structure like folder and subfolders